content/includes/header.html

```diff
@@ -151,4 +151,4 @@
 
 </header>
 
-<script src="/js/usa-search.js"></script>
+<script async src="/js/usa-search.js"></script>
```

Comment:
This script still causes problems because it synchronously loads another script that has a permanent redirect and ends up delaying things by 613ms, according to Chrome's audit.

Reply:
Did you check to see if the login / signup links we be functional on a slower connection?  It is the one thing I am worried about because we fetch the id.me url on load.

Reply:
2 separate discussions here:
1) usa-search.js: @jbalboni, can you open a ticket and assign to me to look into contacting the folks who maintain this? Maybe we can get them to asynchronously load that second script

2) login/signup links (and anything else we're deferring): maybe one more manual QA task to try this out on a slow connection, and if fine, mention that in the PR description (which I will now go re-read in case that's already been done)

Reply:
@raquelromano-gov 

1. Sure. Thinking about it more, I'm not positive that changing it to async is the right course, but we can ask what they think.
2. I've tried it with Chrome's throttling and it works fine for me. This change is unlikely to affect that, since all of the login code is in React and runs after DOMContentLoaded whether the scripts use `defer` or not.

Reply:
Thanks for checking @jbalboni